docs/DeveloperGuide.adoc

```diff
@@ -1,8 +1,9 @@
-= AddressBook Level 3 - Developer Guide
+= NOVA - Developer Guide
```

Comment:
The uml diagrams are not consistent - one is just white boxes but the other is white box highlighted with red lines. Maybe can make them all just non-highlight or make them all highlighted with different colours